cristin-biobank/build.gradle

```diff
@@ -42,4 +39,13 @@ test{
     environment "COGNITO_URI", "https://example.cognito.org"
     environment "CRISTIN_BOT_FILTER_BYPASS_HEADER_NAME", "Hello"
     environment "CRISTIN_BOT_FILTER_BYPASS_HEADER_VALUE", "World"
-}
\ No newline at end of file
+}
+
+//test {
+//    useJUnitPlatform {
+//        excludeTags 'karateTest'
+//    }
+//    environment "COGNITO_URI", "https://example.cognito.org"
+//    environment "CRISTIN_BOT_FILTER_BYPASS_HEADER_NAME", "Hello"
+//    environment "CRISTIN_BOT_FILTER_BYPASS_HEADER_VALUE", "World"
+//}
```

Comment:
When everything is working you should remove commented code

Reply:
done